### DISCUSSION TITLE
我按照文档 restful 2.0 的指导，使用单一向量搜索，其中 searchParams 似乎没有生效，然而在 attu 同样的参数正确。

### DISCUSSION THREAD

**Discussion by 262146150** [milvus-io/milvus#33549] [Q&A and General discussion]
文档中的结构
`{
    "dbName": "string",
    "collectionName": "string",
    "data": [],
    "annsField": "string",
    "filter": "string",
    "limit": "integer",
    "offset": "integer",
    "groupingField": "string",
    "outputFields": [],
    "searchParams": {
        "metricType": "string",
        "params": {
            "radius": "integer",
            "range_filter": "integer"
        }
    },
    "partitionNames": []
}
`

我的调用方法：
`$schema = [
            "collectionName" => 'corp',
            "data" => [$data],
            "annsField" => "vector",
            "filter" => implode(' and ', $filter),
            "limit" => 50,
            "outputFields" => [
                "answer_id",
                'question_id',
                'text'
            ],
            'searchParams' => [
                'metricType' => 'L2',
                'params' => ['radius' => 0.43, 'range_filter' => 0], // Search parameters
            ]
        ];
        $response = $client->post('v2/vectordb/entities/search', [
            'json' => $schema,
        ]);`

得到的数据发现 searchParams  参数没有起任何作用。

**Comment by yhmo** [comment]
PHP？
我不懂php，不过看起来这变量名和值也没啥问题

**Reply by 262146150** [reply]
是的，参数名称没有问题，但是结果并不是预期的
